thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/dashboard/views/tabs/analysis/dimension-tree-map-graph.ftl

```diff
@@ -0,0 +1,49 @@
+<div class="analysis-card analysis-summary bottom-buffer padding-all">
+    <div class="analysis-summary__item">
+      <label class="analysis-summary__label">Current Total</label>
+      <!-- {{displayDate currentStart}} - {{displayDate currentEnd}} -->
```

Comment:
Why comment it out? Is this still needed? If so, please add a comment to clarify. Same for line 9.